app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,76 @@
+@import "foundation_and_overrides";
+@import "index";
+
+.art-details {
+  margin: auto;
```

Comment:
Properties should be ordered background-color, box-shadow, display, margin, margin-bottom, margin-top, width